static/js/client.js

```diff
@@ -116,48 +119,59 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     }
   }
 
-  function generateTitle() {
+  function generateTitle ( ) {
 
     function s(value, sep) { return value ? value + ' ' : sep || ''; }
 
-    var bg_title = '';
+    var title = '';
 
     var time = latestSGV ? latestSGV.mills : (prevSGV ? prevSGV.mills : -1)
       , ago = timeAgo(time, browserSettings);
 
-    if (browserSettings.customTitle) {
-      $('.customTitle').text(browserSettings.customTitle);
-    }
-
     if (ago && ago.status !== 'current') {
-      bg_title =  s(ago.value) + s(ago.label, ' - ') + bg_title;
+      title =  s(ago.value) + s(ago.label, ' - ') + title;
     } else if (latestSGV) {
       var currentMgdl = latestSGV.mgdl;
 
       if (currentMgdl < 39) {
-        bg_title = s(errorcodes.toDisplay(currentMgdl), ' - ') + bg_title;
+        title = s(errorcodes.toDisplay(currentMgdl), ' - ') + title;
       } else {
         var deltaDisplay = delta.calc(prevSGV, latestSGV, sbx).display;
-        bg_title = s(scaleBg(currentMgdl)) + s(deltaDisplay) + s(direction.info(latestSGV).label) + bg_title;
+        title = s(scaleBg(currentMgdl)) + s(deltaDisplay) + s(direction.info(latestSGV).label) + title;
       }
     }
-    return bg_title;  
+    return title;
   }
 
-  function updateTitle(skipPageTitle) {
+  function updateTitle ( ) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/810158318/issues/source?bid=2452379&fileBranchId=2719965#l146)